html/add-product.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/seller.css">
+    <title>Add Product Page</title>
+</head>
+
+<body>
+    <div class="boddy" >
+
+        <div class="main-container">
+            <div class="title">Add Product</div>
+            <form class="add-form" action="#">
+                <div class="product-details">
+                    <div class="input-box">
+                        <label class="details">
+                            Product Name
+                            <input class="data-input" type="text" placeholder=" product Name" required>
+                        </label>
+                    </div>
+                    <div class="input-box">
+                        <label class="details">
+                            Product Image
+                            <input class="data-input" type="text" placeholder=" Image URL" required>
+                        </label>
+                    </div>
+                    <div class="input-box">
+                        <label class="details">
+                            Product Category
+                            <input class="data-input" type="text" placeholder=" Category" required>
```

Comment:
category should be select tag with an options
check https://www.w3schools.com/tags/tag_option.asp